common/persistence/visibility/store/sql/visibility_store.go

```diff
@@ -587,3 +566,77 @@ func (s *VisibilityStore) processRowSearchAttributes(
 	}
 	return searchAttributes, nil
 }
+
+func (s *VisibilityStore) buildQueryStringFromListRequest(
+	request *manager.ListWorkflowExecutionsRequest,
+	executionStatus enumspb.WorkflowExecutionStatus,
+	workflowID string,
+	workflowTypeName string,
+) string {
+	queryTerms := []string{}
+
+	switch executionStatus {
+	case enumspb.WORKFLOW_EXECUTION_STATUS_UNSPECIFIED:
+		queryTerms = append(
+			queryTerms,
+			fmt.Sprintf(
+				"%s != %d",
+				searchattribute.ExecutionStatus,
+				int32(enumspb.WORKFLOW_EXECUTION_STATUS_RUNNING),
```

Comment:
I didn't realized that `ExecutionStatus` is `int`. I changed in in ES, but we can't change here :-(